src/commands/assign.js

```diff
@@ -97,9 +97,37 @@ function setPrompt() {
   }
 
   // Assigned info.
+  //
+  // Shows the number of projects that are currently assigned
   console.log(chalk.green(`Currently assigned: ${chalk.white(assignedCount)}`));
+
+  // Shows assigned projects in a table
+  const submissionDetails = new Table({
+    head: [
+      {hAlign: 'center', content: 'sub'},
```

Comment:
I think the content here should be either `subId` or `sub_id`. I prefer the one with the underscore.